source/includes/steps-fle-create-data-encryption-key.yaml

```diff
@@ -291,8 +357,46 @@ content: |
              status: 0,
              masterKey: { provider: 'local' }
            }
+     .. tab::
+        :tabid: python
+
+        .. code-block:: python
+
+           connection_string = "mongodb://localhost:27017"
+           key_vault_namespace = "encryption.__keyvault"
+
+           fle_opts = AutoEncryptionOpts(
+             kms_providers,
+             key_vault_namespace,
+             mongocryptd_bypass_spawn = True
+           )
+           client = MongoClient(
+             connection_string,
+             auto_encryption_opts = fle_opts
+           )
+           db = client["encryption"]
+           collection = db["__keyVault"]
+           key = collection.find_one({"_id": data_key})
```

Comment:
Nathan & I talked about making this guide a bit more functional, in terms of having the snippets work as their own standalone functions, so that people can copy and paste the code and not be lost in the steps. Given that discussion, I've altered the snippet to make it a function. 

With the new snippet, you can either pass in the data_key as the explicit string or as the actual data_key variable. However, I think having the explicit string doesn't particularly make sense even for java or node. I can't see people explicitly querying for a specific key in their application in a real-world setting. Potentially we could fix the Java & Node examples retroactively in a new ticket.

What do you think?

Reply:
I think we should be consistent in usage. If we use functions/methods only sometimes, what are the specific conditions that warrant them? In the Java code examples, I think the conditions match at least one of the following (open for debate or other ideas):

1) The snippet is a complete, self-contained program, structured idiomatically for that driver language.
2) It is a snippet from a complete, self-contained program taken verbatim.
3) We preface the snippet with "create a function/method".

I think it would be great if we just structured snippets in this step file to follow 2 in which we omit imports and instead depend on them downloading the complete example.

Reply:
We talked about this approach, and I'm ok with it. It just has to be a working example that people can follow along to.

Reply:
For Python, largely everything is wrapped in a function unless it is a snippet to create variables that should be accessible to other snippets (for instance local_master_key is defined outside of any function because we require that later). 

I think this fits under Nathan's requirement of it being a working example that people can follow along with.